diff --git a/node_modules/gamedig/lib/ProtocolResolver.js b/node_modules/gamedig/lib/ProtocolResolver.js
index 9257608..a754b9f 100644
--- a/node_modules/gamedig/lib/ProtocolResolver.js
+++ b/node_modules/gamedig/lib/ProtocolResolver.js
@@ -14,7 +14,7 @@ class ProtocolResolver {
     resolve(type) {
         const path = this.protocolDir+'/'+type;
         if(!fs.existsSync(path+'.js')) throw Error('Protocol definition file missing: '+type);
-        const protocol = require(path);
+        const protocol = require('./protocols/'+type);
         return new protocol();
     }
 }
